helpers/filterSeats.js

```diff
@@ -23,6 +26,12 @@ module.exports = function(dust) {
     var roomRates = params.roomRates;
     var bandColours = params.bandColours;
     var sortByInput = params.sortBy;
+    var seatLegend = params.seatLegend.seatLegend.en;
+
+    var bestPrice = {};
+    var bestSections = [];
+    var currentPrice;
```

Comment:
can probably get away with `var currentPrice, topTicket;` here to create variable that are `null` ready for use later.